README.md

```diff
@@ -1,9 +1,21 @@
 # Pimcore Admin Classic Bundle
 
+> [!IMPORTANT]
+> This bundle has reached **end of feature development.** We will no longer
+
+> accept or review any pull requests related to new features or improvements for this bundle.
+>
+> However, bug fixes will continue to be accepted, reviewed, and merged at least until the end of 2025.

```

Comment:
```suggestion
> However, we will continue to accept, review and merge bug fixes at least until the end of 2025.

```
A bit petty, but it might be better to restructure this sentence a bit.